src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -474,41 +384,15 @@ public int NotifyAfter
                 }
                 else
                 {
-                    throw ADP.ArgumentOutOfRange("NotifyAfter");
+                    throw ADP.ArgumentOutOfRange(nameof(NotifyAfter));
                 }
             }
         }
 
-        internal int ObjectID
-        {
-            get
-            {
-                return _objectID;
-            }
-        }
-
-        /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml' path='docs/members[@name="SqlBulkCopy"]/SqlRowsCopied/*'/>
-        public event SqlRowsCopiedEventHandler SqlRowsCopied
```

Comment:
It's a public API! Was it removed accidentally?

Reply:
It was moved to the top of the file as an autogenerated event instead of being down here, the xml comment was moved with it.